src/api/index.ts

```diff
@@ -27,6 +30,18 @@ const EARN_SUPPORTED_POSITION_IDS = new Set([
   `${NetworkId['celo-mainnet']}:0xfb2c7c10e731ebe96dabdf4a96d656bfe8e2b5af`,
 ])
 
+i18next
+  .use(Backend)
+  .use(i18nextMiddleware.LanguageDetector)
+  .init({
+    backend: {
+      // eslint-disable-next-line no-path-concat
+      loadPath: '../../locales/{{lng}}.json',
+    },
+    fallbackLng: 'en',
```

Comment:
it looks like english is fallback language for the wallet, should we do the same here?